src/components/ReportActionItem/MoneyRequestAction.js

```diff
@@ -66,11 +78,40 @@ const defaultProps = {
     iouReport: {},
     reportActions: {},
     isHovered: false,
+    session: {
+        email: null,
+    },
 };
 
 const MoneyRequestAction = (props) => {
     const hasMultipleParticipants = props.chatReport.participants.length > 1;
     const onIOUPreviewPressed = () => {
+        // This would ideally be passed as a prop or hooked up via withOnyx so that we are not be triggering a potentially intensive
+        // search in an onPress handler, I think this could lead to performance issues but it probably ok for now.
+        const thread = getThreadForReportActionID(props.action.reportActionID);
+        console.log({thread});
+        if (_.isEmpty(thread)) {
+            // Since a thread does not exist yet then we need to create it now. This is done by creating the report object
+            // and passing the parentReportActionID of the reportAction. OpenReport will then automatically create the thread for us.
+            const optimisticThreadReport = buildOptimisticChatReport(
+                props.chatReport.participants,
+                props.translate('iou.threadReportName', {payee: props.action.actorEmail, comment: props.action.originalMessage.comment}),
+                '',
+                props.chatReport.policyID,
+                props.chatReport.owner,
+                props.chatReport.type === CONST.REPORT.CHAT_TYPE.POLICY_EXPENSE_CHAT,
```

Comment:
This should be `props.chatReport.isOwnPolicyExpenseReport`, no?

Reply:
ah yeah makes sense